lib/ansible/modules/network/cnos/cnos_vlan.py

```diff
@@ -23,532 +21,398 @@
 # Module to send VLAN commands to Lenovo Switches
 # Overloading aspect of vlan creation in a range is pending
 # Lenovo Networking
-#
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
-                    'supported_by': 'community'}
-
+                    'supported_by': 'network'}
 
-DOCUMENTATION = '''
+DOCUMENTATION = """
 ---
 module: cnos_vlan
-author: "Anil Kumar Muraleedharan (@amuraleedhar)"
-short_description: Manage VLAN resources and attributes on devices running
- Lenovo CNOS
+version_added: "2.8"
```

Comment:
I'm not sure why you're changing `version_added`? Is that because of the hard break on interface?

Reply:
I will revert it into 2.3